sticker-graph/sticker_graph/model.py

```diff
@@ -24,21 +24,21 @@ def affine_transform(self, prefix, x, n_outputs):
         input_size = x.get_shape()[-1]
         batch_size = tf.shape(x)[0]
 
-        w = tf.get_variable(
+        w = tf.compat.v1.get_variable(
             "%s_linear_w" % prefix, [
                 input_size, n_outputs])
-        b = tf.get_variable("%s_linear_b" % prefix, [n_outputs])
+        b = tf.compat.v1.get_variable("%s_linear_b" % prefix, [n_outputs])
 
         x = tf.reshape(x, [-1, input_size])
 
         return tf.reshape(
-            tf.nn.xw_plus_b(
+            tf.compat.v1.nn.xw_plus_b(
                 x, w, b), [
                 batch_size, -1, n_outputs])
 
     def decode_word(self, word, max_subword_len):
-        bytes = tf.decode_raw(word, tf.uint8)[:max_subword_len]
-        return tf.pad(bytes, [[0, max_subword_len - tf.shape(bytes)[0]]])
+        bytes = tf.io.decode_raw(word, tf.uint8)[:max_subword_len]
```

Comment:
Unrelated, but isn't `bytes` a built-in?

Reply:
Not a keyword:
https://docs.python.org/3.7/reference/lexical_analysis.html#keywords

Which is probably why this is permitted. But it is indeed a built-in class, so this should probably be renamed.